src/components/TokenEnterAmount.tsx

```diff
@@ -381,81 +410,95 @@ export default function TokenEnterAmount({
         </View>
       </Touchable>
       {token && (
-        <View
-          style={[
-            styles.rowContainer,
-            { borderTopLeftRadius: 0, borderTopRightRadius: 0, borderTopWidth: 0 },
-          ]}
-        >
-          <TextInput
-            forwardedRef={inputRef}
-            onChangeText={(value) => {
-              handleSetStartPosition(undefined)
-              onInputChange(value)
-            }}
-            value={formattedInputValue}
-            placeholderTextColor={Colors.gray3}
-            placeholder={amountType === 'token' ? placeholder.token : placeholder.local}
-            keyboardType="decimal-pad"
-            // Work around for RN issue with Samsung keyboards
-            // https://github.com/facebook/react-native/issues/22005
-            autoCapitalize="words"
-            autoFocus={autoFocus}
-            // unset lineHeight to allow ellipsis on long inputs on iOS. For
-            // android, ellipses doesn't work and unsetting line height causes
-            // height changes when amount is entered
-            inputStyle={[
-              styles.primaryAmountText,
-              inputStyle,
-              Platform.select({ ios: { lineHeight: undefined } }),
+        <View>
```

Comment:
This addition/removal is purely a result of adding extra `View` wrapper around the input as it is needed for the new `loading` check below. The actual TextField props are unchanged.